deploy/bin/setup-ee.sh

```diff
@@ -74,17 +79,17 @@ if ! ./deploy/bin/make-aws-secret.sh; then
 fi
 
 # Configmaps, secrets, and deployments
-$K delete configmap --ignore-not-found edge-config -n ${DEPLOYMENT_NAMESPACE}
```

Comment:
The `-n namespace` flags are all redundant because it's built into `$K`.  Leaving it in is confusing.